packages/elements/tsup.config.ts

```diff
@@ -21,7 +21,7 @@ export default defineConfig(overrideOptions => {
       'react/sign-in/index': 'src/react/sign-in/index.ts',
       'react/sign-up/index': 'src/react/sign-up/index.ts',
     },
-    external: ['react', 'react-dom', '@statelyai/inspect'],
+    external: ['react', 'react-dom', 'next', '@statelyai/inspect'],
```

Comment:
While https://tsup.egoist.dev/#excluding-packages explains

> but dependencies and peerDependencies in your package.json are always excluded

I guess it doesn't hurt explicitly adding it here if we have `react` and `react-dom` here already